test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectConfigurationRetrieverTests.cs

```diff
@@ -28,7 +30,13 @@ public async Task FromNetwork()
         [Fact]
         public async Task FromFile()
         {
-            var context = new CompareContext();
+            var context = new CompareContext
+            {
+                PropertiesToIgnoreWhenComparing = new Dictionary<Type, List<string>>
```

Comment:
Would be great to add comments explaining why this property should be ignored when comparing.